web/conf/start.sh

```diff
@@ -2,9 +2,24 @@
 # Copyright (C) 2021 Monocle authors
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
+# so we don't use anything in /app that doesn't have permission
+temp_file=$(mktemp)
+# this does basically what `sed -i` does, just without writing a
+# intermedeiary file in the target folder
+# with `sed -i` the output is:
+# "sed: couldn't open temporary file XXX: Permission denied"
+sed_temporal() {
+  local expression=$1
+  local target_file=$2
+  sed -e "${expression}" "${target_file}" > "${temp_file}"
+  cp "${temp_file}" "${target_file}"
```

Comment:
you might have used mv instead of cp/rm.

Reply:
I remember mv not working for some reason, but I am not sure